lib/web/plug.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Xcribe.Web.Plug do
```

Comment:
```suggestion
defmodule Xcribe.SwaggerPlug do
```

:nail_care: As it is something specific to swagger, WDYT of naming to make that clear.

Reply:
Currently we just support Swagger but I intend to support Api Blueprint soon. And probably we'll use the same plug and choose the template for configured format.